liveclasses/views.py

```
@@ -0,0 +1,53 @@
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        course = get_object_or_404(Course, slug=self.kwargs['slug'])
+        context['course'] = course
+        context['teacher'] = course.teachers.filter(teacher=self.request.user).exists()
+        return context
+
+
+class CreateLiveClassView(LoginRequiredMixin, CreateView):
+    model = LiveClass
+    template_name = "liveclasses/create_live_class.html"
+    fields = ['topic', 'scheduled_at', 'duration']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)